src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource_formatters.py

```diff
@@ -0,0 +1,424 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+
+from azure.mgmt.resource.resources.models import (
```

Comment:
We also need to use get_model() here.

Reply:
Replaced `import` with `get_model()`. Regarding using `mock()`, I don't think it's needed in the formatters unit tests (since there's no dependencies to assert/verify) but I added two tests with mocks to verify prompt logic with `-confirm-with-what-if`, which seems to be worth testing to me.